nest/00. Foreword.md

```diff
@@ -0,0 +1,24 @@
+# Getting started with Nest
+
+## Foreword
+
+Welcome to Nest's official getting-started guide! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.
+
+In order to avoid any confusion, we will refer to the project as a whole as `Nest` (with an upper case n), while we will refer to the binary as `nest` (with a lower case n). Thus, `finest` refer to its eponymous binary, and is included in Nest.
+
+[//]: # (TODO: add link to the installation instructions page of Raven-OS)
+Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
+
+## How to read this guide
+
+nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. That latter is targeted towards more advanced users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.
```

Comment:
```suggestion
nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. The latter is targeted towards more experimented users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.
```
Proper English:
That later -> The later
Better flow:
advanced users -> experimented